pallets/credit/src/tests.rs

```diff
@@ -82,6 +82,103 @@ fn update_credit_setting() {
     });
 }
 
+#[test]
+fn update_credit_data() {
+    new_test_ext().execute_with(|| {
+        let credit_setting = CreditSetting {
```

Comment:
Just a minor thing: since we have `CreditSettings` set up in `new_test_ext()` function, we don't actually need the code to update credit setting in the `update_credit_data()` test. We can probably remove the credit setting code to make the `update_credit_data` unit test "purer"

Reply:
I will remove the credit setting code in `update_credit_data` unit test.